RoadCare3/FormLogin.cs

```diff
@@ -377,7 +376,17 @@ private void FormLogin_Load(object sender, EventArgs e)
             chkUseIntegratedSecurity.Checked = Settings.Default.USE_INTEGRATED_SECURITY;
 
             tbRoadCareUserName.Text = Settings.Default.LAST_LOGIN;
-        
+            TabSqlServer.Enabled = true;
+            #if DEBUG
```

Comment:
I don't think its a good idea to have this stuff in the form code. Someone can simply grab the database password from IL-Spy. This should be moved to a config file which would ideally be encrypted.